packages/lib/slots.ts

```diff
@@ -163,11 +163,11 @@ function buildSlotsWithDateRanges({
   const slots: { time: Dayjs; userIds?: number[] }[] = [];
 
   dateRanges.forEach((range) => {
-    const startTimeWithMinNotice = dayjs.utc().add(minimumBookingNotice, "minute");
```

Comment:
Again this being in UTC might cause issue with availability interval due to change in value of minutes.